app/controllers/api/plans_base_controller.rb

```diff
@@ -112,6 +112,18 @@ def plans_index_path
 
   protected
 
+  def new_masterize_plan(issuer, assoc)
```

Comment:
Are we intending to use this method in another controller the inherits from `Api::PlansBaseController`?

Reply:
Not that I am aware of.